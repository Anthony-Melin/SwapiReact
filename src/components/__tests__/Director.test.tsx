import { describe, it, expect, beforeEach } from 'vitest'
import { render, screen } from '@testing-library/react'
import Director from '../Director.tsx'

describe('Director', () => {
    beforeEach(() => {
        render(<Director value="VALUE" />)
    })

    it('should match snapshot', () => {
        expect(screen.getByTestId('director')).toMatchSnapshot()
    })

    it('should have text', () => {
        expect(screen.getByTestId('director')).toHaveTextContent(
            'Director: VALUE'
        )
    })
})
