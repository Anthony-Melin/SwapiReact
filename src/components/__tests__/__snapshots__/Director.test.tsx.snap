// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Director > should match snapshot 1`] = `
<p
  data-testid="director"
>
  Director
  : 
  VALUE
</p>
`;
